spec.html

```diff
@@ -40775,11 +40775,15 @@ <h2>Syntax</h2>
           `\` `B`
           [+U] `(` `?` `=` Disjunction[+U, ?N] `)`
           [+U] `(` `?` `!` Disjunction[+U, ?N] `)`
+          [+U] `(` `?` `&lt;=` Disjunction[?U, ?N] `)`
+          [+U] `(` `?` `&lt;!` Disjunction[?U, ?N] `)`
           [~U] QuantifiableAssertion[?N]
 
         QuantifiableAssertion[N] ::
           `(` `?` `=` Disjunction[~U, ?N] `)`
           `(` `?` `!` Disjunction[~U, ?N] `)`
+          `(` `?` `&lt;=` Disjunction[?U, ?N] `)`
+          `(` `?` `&lt;!` Disjunction[?U, ?N] `)`
```

Comment:
It's an error to use '?U' here, because QuantifiableAssertion doesn't have a 'U' parameter.